_includes/footer.html

```diff
@@ -0,0 +1,88 @@
+{% assign identifier = site.data.usa_identifier.identifier_data[0] %}
+
+<footer class="usa-class">
+  <section>
+    <div class="usa-footer__secondary-section"> <div class="grid-container">
+      <div class="usa-footer grid-row">
+        <div class="grid-col-auto">
+          This project is maintained by <a class="usa-link" href="https://18f.gsa.gov/">18F</a>
```

Comment:
Same little nitpick as before, if we want to use the data file for populating the link